Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -115,11 +109,24 @@ - (BOOL)managedObjectContext:(NSManagedObjectContext *)context
     return result;
 }
 
-- (void)mainThreadExtraInfo:(SentrySpan *)span
+- (void)addExtraInfoToSpan:(SentrySpan *)span withContext:(NSManagedObjectContext *)context
 {
     BOOL isMainThread = [NSThread isMainThread];
 
     [span setDataValue:@(isMainThread) forKey:BLOCKED_MAIN_THREAD];
+    NSMutableArray<NSString *> *systems = [NSMutableArray<NSString *> array];
+    NSMutableArray<NSString *> *names = [NSMutableArray<NSString *> array];
+    [context.persistentStoreCoordinator.persistentStores enumerateObjectsUsingBlock:^(
+        __kindof NSPersistentStore *_Nonnull obj, NSUInteger idx, BOOL *_Nonnull stop) {
+        [systems addObject:obj.type];
+        if (obj.URL != nil) {
+            [names addObject:obj.URL.path];
+        } else {
+            [names addObject:@"(null)"];
```

Comment:
h: I believe we need a better option to fallback. This is not user friendly and I'm foreseeing complains.

Reply:
I'm open to any alternatives. Nobody has proposed any so far.

Reply:
I would also be interested to know the cases where the URL is nil. I expected it to be nil for in memory stores, but even that has a value like `memory://563428497`. It is probably nil if there is no persistent store/coordinator at all, but then I don't know what the complaint would be.

Reply:
> I'm open to any alternatives.

`<App Name>:coredata` anything really other than (null)

Reply:
We already gather the equivalent of that in the `type`. 

In this case, we should send some kind of sentinel value so the frontend knows to render a special state equivalent to "there was no database". This seems as good as any.

Reply:
I agree with @armcknight here - in case this occurs a lot we can always re-iterate.